src/grpc/iot_config/hpr_route_ets.erl

```diff
@@ -556,13 +542,25 @@ skfs_count_for_route(RouteID) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
--spec do_remove_skf(ets:table(), hpr_skf:skf()) ->
-    DeletedCount :: non_neg_integer().
+%% Table is `ordered_set' with the first being a negated timestamp. This gives
+%% us a sort order of most-recently-used. `{keypos, 2}' makes the table unique
+%% on the second key.
+%% Structure: {Timestamp, SessionKey, DevAddr, MaxCopies}.
+-spec mk_new_skf_ets(hpr_route:id()) -> ets:table().
+mk_new_skf_ets(RouteID) ->
+    ets:new(?ETS_SKFS, [
+        public,
+        ordered_set,
+        {read_concurrency, true},
+        {write_concurrency, true},
+        {keypos, 2},
```

Comment:
idk if this will work as the `ordered_set` will be on the key. Can you 2x check?

Reply:
> one`` key, one object, ordered in Erlang term order

Maybe it's fine